pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
 
   #TS and Modeling libraries
   "statsmodels >=0.12.1",
+  "sktime",
```

Comment:
we should not have sktime as a dependency in an ideal world, created a ticket for it (but the basic_report() is dependent on it)